src/material-examples/datepicker-disabled/datepicker-disabled-example.html

```diff
@@ -18,6 +18,6 @@
   <mat-form-field>
     <input matInput [matDatepicker]="dp3" placeholder="Input disabled" disabled>
     <mat-datepicker-toggle matSuffix [for]="dp3"></mat-datepicker-toggle>
-    <mat-datepicker #dp3 disabled="false"></mat-datepicker>
+    <mat-datepicker #dp3 [disabled]="false"></mat-datepicker>
```

Comment:
The old usage was also valid.

Reply:
Yes, however shouldn't we prefer to pass the literal `false` instead of `string` with `false` value?

Reply:
Since we handle both strings and booleans, and the value doesn't change, it would be better to use the string binding. [See the template syntax docs](https://angular.io/guide/template-syntax#one-time-string-initialization).